src/Sulu/Component/Content/Types/ResourceLocator/Strategy/ResourceLocatorStrategy.php

```diff
@@ -90,7 +90,7 @@ public function __construct(
         $this->resourceLocatorGenerator = $resourceLocatorGenerator;
     }
 
-    public function generate($title, $parentUuid, $webspaceKey, $languageCode, $segmentKey = null)
+    public function generate($title, $parentUuid, $webspaceKey, $languageCode, $segmentKey = null, $uuid = null)
```

Comment:
Same as above should use something like here for BC promise: https://github.com/sulu/sulu/blob/ddf5a0842807aa649c97bccf8245fe6e621d81cb/src/Sulu/Bundle/MediaBundle/Media/ImageConverter/MediaImageExtractor.php#L50-L61